pay-stamp-duty-land-tax/_footer.html

```diff
@@ -44,8 +44,8 @@ <h2 class="visuallyhidden">
               <time datetime="2016-04-01T08:00:06.000+01:00" class="timestamp">1 April 2016</time>
               <strong>8:06am</strong><br />
               Credit card fees have changed.
-              <br/><a href="./v3.2.html">View this version</a>
-              <br/><a href="./v3.2-diff.html">Compare this version to current version</a>
```

Comment:
By the way - these numbers reflected the major minor aspect. eg that was 2 minor updates to the 3rd major update. 

This is reflected somewhat in the data that was pulled out [here](https://gist.github.com/kevindew/68897fe78f07145c5fc619595bcc09d6) 

Since all the prototypes seem to have gone down a x.0.html approach though we can probably just clear off the .0 from all of them.